src/ESPEasyWiFiEvent.cpp

```diff
@@ -113,7 +113,7 @@ void WiFiEvent(system_event_id_t event, system_event_info_t info) {
     case SYSTEM_EVENT_ETH_START:
       addLog(LOG_LEVEL_INFO, F("ETH Started"));
       char hostname[40];
```

Comment:
Don't create objects here in a case.
If you need to create objects when handling a case, then you _must_ add scope braces.

```c++
case bla:
{
  // do your stuff
  break;
}
```

Reply:
OK, I'll change it.